code/api/tests/test_inbox.py

```diff
@@ -223,6 +223,9 @@ def test_post_comment_local_like(self):
         self.assertEqual(response.status_code, 200)
         self.assertEqual(comment.total_likes(), 1)
         liker = comment.likes.all()[0]
+        temp = list(Author.objects.all())
+        a = liker.author
+        b = author1
```

Comment:
just some temp variables to remove

Reply:
Oops thanks for catching that, thought I had them all